Media download rework

jfwallin opened the pull request.

Sharkful committed:
Changed media System to Use downloaded Zip File

LoginManager now downloads a zip file instead of just a JSON after a lab is selected
DownloadUtility now has a function to download and extract a zip file, preserving subdirectories
Login Manager now initializes MediaCatalogue by giving it the location of the downloaded resources folder
MediaCatalogue now Initializes itself by reading through the subdirectories and loading the resources into Unity
Added a library with a csc.rsp file to help with zip extracting
Modified MediaInfo and MediaPlayer classes to work with new resource handling system.

Sharkful committed:
Debugging for cemo 10 running

Sharkful committed:
More Bridge debugging

Changed ObjectInfo default parent object to "[CURRENT_LAB]" from "[_DYNAMIC]"
Changed Bridge to use the Media catalogue
Changed demo10.json to be inside of a lab json
changed texture names in demo10.json to not include subfolder name
Added Logger calls in Bridge
bridge now adds text component if it can't find one

Sharkful committed:
Web Loading with the Bridge is Running

Added some extra logs to the bridge to help determine where content is placed and how big it is
Added more logs to demoSequence, lightingControl, and demo to track application flow
DownloadUtility Now checks if a file already exists before downloading/extractin, and deletes it.
this avoids an error when extracting a file that already exists
Changed what the Login Manager sets the downloaded resource zip and folder name to when the download is
from a debug endpoint. May need to change it for production endpoints in the future as well.

Sharkful committed:
Tweaks and bug Squashing

added position tracking code to LabLogger, writes binary data to extra file
Changed log filename to include user's name
Fixed the guest login button
Changed the bridge logging for mesh extents

Sharkful committed:
Final Web Moon Lab Demo build

Various last minute fixes:
Changed permissions manifest to include internet access (don't click synchronize)
changed the sun material to be less washed out
Changed position log filename
fixed position log upload
Directional light now moves to [CURRENT_LAB] after placement, is no longer childed to the camera
[CURRENT_LAB] now moves to the location of the placement object once it is placed
and many more

nealmcclain commented:
Looks like you've been working!

Also looks like I'm still on this mailing list and probably shouldn't be.  :-)  I don't mind getting the emails though so whatever you want to do is fine.

   neal
________________________________
From: John Wallin ***@***.***>
Sent: Thursday, June 30, 2022 8:26 PM
To: jfwallin/cyberAR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [jfwallin/cyberAR] Media download rework (PR #3)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/jfwallin/cyberAR/pull/3<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9ZggwSh0$>

Commit Summary

  *   7203666<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/commits/7203666d19a50ba4bc3ecaf0a6acaf2fec16a82b__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9jhP0HKw$> Changed media System to Use downloaded Zip File
  *   8857619<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/commits/8857619ef85d90237083e7a8102a59d64a76c301__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9C70CXe8$> Debugging for cemo 10 running
  *   203db9d<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/commits/203db9d593afdc2945126e336c5edb0bfc9d6b03__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9bb6Thkk$> More Bridge debugging
  *   991c5ef<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/commits/991c5efc052e42ef5bab5ab8e19e8d5f382617b8__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9c3mfSY8$> Web Loading with the Bridge is Running
  *   f4b810d<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/commits/f4b810d22fe788691544d068a3509010c1fe62b9__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9IqhFAjM$> Tweaks and bug Squashing
  *   c27e2dc<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/commits/c27e2dcb2ae7f3b5bb07db551c3c6441c3f8a0a0__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU955Ure6A$> Final Web Moon Lab Demo build

File Changes

(35 files<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9m-20Sns$>)

  *   A Untitled.mlscheme<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-3986b79ad2fcc9a3d9d6d018cf17f19e93df04a528367917577b5e70c82c3c37__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9Pck3B_E$> (286)
  *   M _Code Device/AR Labs/Assets/JSON Bridge/Scripts/Bridge.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-a3b6a9da2839caa079e7832b33f2671734f2cb3da7a9b924ec7c4ea635b360a3__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9a-zw6qA$> (28)
  *   M _Code Device/AR Labs/Assets/JSON Bridge/Scripts/ObjectInfo.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-30183a10f450cc6ca1ae692a6554f404eee7e5611f05638711609f0d86fecf47__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9d4szWH0$> (2)
  *   M _Code Device/AR Labs/Assets/Login Scene/loginLogic.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-d5e41cfe41c96e3ad59a6917b40536eb8db09cff911f3989c960d3d1e667c5b6__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9EAcn9Hg$> (4)
  *   M _Code Device/AR Labs/Assets/Plugins/Lumin/MagicLeapManifestSettings.asset<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-7fe17bf8a24eb052e28016969919f5c25855f785952178afdf4771267cc9a79a__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9STl3xas$> (27)
  *   M _Code Device/AR Labs/Assets/Resources/Materials/_k_sun.mat<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-dde67e3bceab3ec81e5863e0c54ca0fdfd49f92729df2939fb7fb0aec4fdbb58__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9FZ12gos$> (4)
  *   M _Code Device/AR Labs/Assets/Resources/Prefabs/demoPrefab.prefab<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-22969dd1bb05b5525dfcaff38e0eb21d808ab828add79994df551489a45c5111__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9g3kmgr8$> (13)
  *   D _Code Device/AR Labs/Assets/Resources/Textures/balldimpled.png<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-ce61abf9283177794f6f6b6ac8e2313c0a532935e25f36b5f71929a90aec9eed__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9feTEogA$> (0)
  *   D _Code Device/AR Labs/Assets/Resources/Textures/balldimpled.png.meta<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-0ac3a55afff1e11ce5f02474587930a4860ed14a19ec3c7bcd792acefde80396__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9YcthNZI$> (94)
  *   A _Code Device/AR Labs/Assets/Resources/jsonDefinitions/MoonMCQ.json<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-5157d45d152007a7fc3c8f81afe31c0054ec64814be859fa613596b59c1b5414__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU91CidVug$> (35)
  *   A _Code Device/AR Labs/Assets/Resources/jsonDefinitions/MoonMCQ.json.meta<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-8ea4de47bb863f6cff23d249777d877517c20d21d380dc48e09b2572434fac3e__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9dVpRsro$> (7)
  *   M _Code Device/AR Labs/Assets/Scenes/In Development/New Login.unity<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-6099f385cc30fa69ccc82d37929f0cf4845f1487cc32ebeb35cec0aa8d10b67a__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9Ni9SsYk$> (506)
  *   M _Code Device/AR Labs/Assets/Scenes/In Development/newlogintes.unity<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-744f01c033510759a61c4219255783fbf12b1dfad9c879567bbde7f01780a7bb__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9Xxw0iKk$> (33)
  *   M _Code Device/AR Labs/Assets/Scripts/Activity Modules/Demo/demo.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-4d9a716b952152054ac3476a522675d00d95d1416e23db165ef3251768664027__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU90FNGO7A$> (8)
  *   M _Code Device/AR Labs/Assets/Scripts/LabControl/LabControl.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-a5c327a7c1d2809a1b0c15723c1dacdaf4c0170eee8242cc2e5811df9866cd1f__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9Hkbls3o$> (2)
  *   M _Code Device/AR Labs/Assets/Scripts/LabControl/Rewrite/LabManager.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-e4b08114f0ac6a47cb6cde3da8468c1a67cc12f467ed1b65f3b26f1099d2a2d7__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9dnhSa2Y$> (4)
  *   M _Code Device/AR Labs/Assets/Scripts/LoginRestructure/Authenticate.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-1a7109d69ccacb2338240fdacec0ab5033e113531c096f7c0f7ab310f4a0d1b6__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU94rHEPu4$> (5)
  *   M _Code Device/AR Labs/Assets/Scripts/LoginRestructure/DownloadUtility.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-3b3f6636d84e80e60c6706cdb91e4a1204512f2f0d04c55b9ae86b3b124b2332__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9_ynn-YQ$> (102)
  *   A _Code Device/AR Labs/Assets/Scripts/LoginRestructure/Editor/LoginManagerEditor.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-8dbde2ff153c48d87c88d942aeb9f1ad2c9d5068f69e1f979d3f745310a5cf6f__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9rCX8Gc8$> (26)
  *   A _Code Device/AR Labs/Assets/Scripts/LoginRestructure/Editor/LoginManagerEditor.cs.meta<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-70adcb01c21e71bd7b4b085cee0c1c75af1656593840b10cf44f6a5a9e3ae314__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9c8Z8lmM$> (11)
  *   M _Code Device/AR Labs/Assets/Scripts/LoginRestructure/LabLogger.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-6c353f0420bbc5251e272d93135dd3bea9ebc231f1e18005a734c39445ae6f22__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU90mpBAIc$> (89)
  *   M _Code Device/AR Labs/Assets/Scripts/LoginRestructure/LoginManager.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-2c39636a179210bd0164feacd63b87c358eff5e2a0d88e88d9ba6975400b6cf1__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9yWQOLKk$> (166)
  *   M _Code Device/AR Labs/Assets/Scripts/MediaManager/MediaCatalogue.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-35dd97e76d83c3e2ef31a82eca3ba4f277c68693aa8c21f271dcaba64981d6bf__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU91YBMBCU$> (257)
  *   M _Code Device/AR Labs/Assets/Scripts/MediaManager/MediaInfo.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-162bd3e25550b8e1b211ff3baa08ad4aa6e4cb256159e1b85ea1f75e4249381c__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9jNR3LdI$> (3)
  *   M _Code Device/AR Labs/Assets/Scripts/MediaPlayer/MediaPlayer.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-109f7eace107bc563bc6f75631f99f464f1d8ce13badaa302dc73e1e409ac4b7__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9ishQGws$> (10)
  *   M _Code Device/AR Labs/Assets/Scripts/ScriptSerializer/Serializer.asset<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-6f15f6913ba27bca2d387eecdcd287920307d9ed2a998cf3ca654594a4bc8eb4__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9qBUhXzQ$> (2903)
  *   M _Code Device/AR Labs/Assets/Scripts/_childObjectScripts/lightingControl.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-85fac7ab47eb4b38ce0029b55e036bda43f5e8492d3b8315a99363e5fea42fcb__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9M6ZtDho$> (12)
  *   M _Code Device/AR Labs/Assets/Scripts/demoSequences/demoSequence.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-b837c5145a63811a60597d4e184d0f64a0a451a55e0decc90230141d047ef8d5__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9QJ69OcU$> (4)
  *   A _Code Device/AR Labs/Assets/ZipTester.cs<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-d9b5c1f213c272e81d193c60fd1677c39d946aee320d4168544bf26cdc01859c__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU90gg8Slg$> (50)
  *   A _Code Device/AR Labs/Assets/ZipTester.cs.meta<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-d4727bb2dc5cfbe41ff8cbe7bc98335b3fb6f067f3745d8d7ac38a2f702f14ac__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9ung1dao$> (11)
  *   A _Code Device/AR Labs/Assets/csc.rsp<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-df22469f3286c38615673c7e610cff1a355fb48e5762c7a613901f214b2463f1__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9MjXsBJc$> (2)
  *   A _Code Device/AR Labs/Assets/csc.rsp.meta<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-eac8139fafc9e029a5f100d50b8ba5d1f49725e37f5dead97425d232dabd9f2e__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9Pb-iN1o$> (7)
  *   M _Code Device/AR Labs/Packages/packages-lock.json<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-a1bae7156bc625a2dc1775da2d90453418fc527ec60e4b753b55ebba2c28f576__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9UD33XME$> (2)
  *   M _Code Device/AR Labs/ProjectSettings/ProjectSettings.asset<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-3d82385c3adb54289cdc7b64cd85a640134d16172cd3b30da396fe449561f02d__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9BZ6yeww$> (7)
  *   M _Code Device/AR Labs/UserSettings/EditorUserSettings.asset<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3/files*diff-855a43b48dea5e4a9371f4577556531e76ccb1063c026cca83e6d979c48347e9__;Iw!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9-utPJNw$> (4)

Patch Links:

  *   https://github.com/jfwallin/cyberAR/pull/3.patch<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3.patch__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9mx_jroo$>
  *   https://github.com/jfwallin/cyberAR/pull/3.diff<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3.diff__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9SIZfF7I$>

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/jfwallin/cyberAR/pull/3__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9ZggwSh0$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/ACCBSCZVNK5J5RNKSC7MQVTVRZCMFANCNFSM52LEVBUQ__;!!Cx0GcAwIk107!-DmRH1JupXMkqZQMUfhuQ1ixQWveUlZqamwOvNhKimQXAkd1YY0HoeOokVwVCH0x8v0XBW_vc8BZgCMtgKU9SkxEcy0$>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Sharkful commented:
Added transform tracking fix the lack of precise timing information

Sharkful committed:
Transform Tracking Fixes

Added more precise time tracking to the transform logger.
It now logs the start time to sync with the primary log, as well as the offsets from 0.5s at each log.
